app/models/timestamp.rb

```diff
@@ -34,6 +34,8 @@ class Timestamp
   class Exception < StandardError; end
 
   class TimestampParser
+    DURATION_REGEX = /[+-]?P/ # ISO8601 "Period"
```

Comment:
I feel this constant could make the comment obsolete if it's named something along the lines of

- `ISO8601_DURATION_REGEX`
or
- `ISO8601_PERIOD_REGEX`

It'd read nicely to me :)